### ISSUE TITLE
Error `ps:set-restart-policy discourse always` is not a dokku command

### ISSUE THREAD

**Issue by whollacsek** [badsyntax/dokku-discourse#10]
I'm getting this error with dokku version 0.22.0

```
 !     `ps:set-restart-policy discourse always` is not a dokku command.
 !     See `dokku help` for a list of available commands.
```

This is a breaking change from dokku: http://dokku.viewdocs.io/dokku~v0.22.0/deployment/process-management/#restart-policies

You may remove this command https://github.com/badsyntax/dokku-discourse/blob/251a73f4b2687971428c5c511cf83ee9e90612fd/internal-functions#L190 
Since
> By default, Dokku will automatically restart containers that exit with a non-zero status up to 10 times via the on-failure Docker restart policy.

**Comment by badsyntax** [comment]
I've released the fixes with version 0.2.2: https://github.com/badsyntax/dokku-discourse/releases/tag/0.2.2

Are you able to verify the fixes at all?

You can update the plugin with `dokku plugin:update discourse 0.2.2`

**Comment by whollacsek** [comment]
I haven't had the time to run the setup again but from the look for the changes you added we can close this for now, thank you!
